pygmt/src/grdcontour.py

```diff
@@ -28,7 +28,7 @@
     t="transparency",
 )
 @kwargs_to_strings(
-    R="sequence", L="sequence", A="sequence_plus", c="sequence_comma", p="sequence"
+    R="sequence", L="sequence", A="sequence", c="sequence_comma", p="sequence"
```

Comment:
We want to convert `annotation=[100, 200, 300]` to `A="100,200,300"`, so we should use `A="sequence_comma"` instead, but it won't work for the case of a single contour (e.g., `annotation=[100]`). It will convert `annotation=[100]` to `A="100"`, but we expect it to be `A="100,"` (not the trailing comma). I think we need to remove `A="sequence"` and add codes like:
```
if is_nonstr_iter(kwargs.get("A")):
    if len(kwargs["A"]) == 1:
        kwargs["A"] = str(kwargs["A"]) + "," 
    else:
        kwargs["A"] = ",".join(f"{item}" for item in kwargs["A"])
```